### ISSUE TITLE
Support explicit filetype passing in TinyTag.get()

### ISSUE THREAD

**Issue by wjanssen** [devsnd/tinytag#84]
Some music tools, like Python Audio Tools, leave FLAC files in a directory with an extension-less 8-character filename if the constructed filename contains bad characters.  It would be great if tinytag would support these by allowing one to explicitly pass a filename extension into "get()", which it would then pass to "_get_parser_for_filename()".

I would also suggest using os.path.splitext() to obtain the extension in "_get_parser_for_filename()"

**Comment by devsnd** [comment]
Hey @wjanssen,

For the time being you can use the parser class directly, e.g. 
```
filename = 'something_without_file_extension'

from tinytag import Flac
import os
import io

with io.open(filename, 'rb') as af:
            size = os.path.getsize(filename)
            tag = Flac(af, size, ignore_errors=ignore_errors)
            tag.load(tags=True, duration=True)
```

I guess the nicest solution would be to be able to determine the filetype by it's file header instead. I opened a separate ticket for that suggestion here #85 

But I guess your use-case still warrants a slight refactoring, I'll look into it.

**Comment by devsnd** [comment]
In the `master` branch you can find a version of tinytag that can automatically determine the filetype by the file's magic header bytes. That should fix your issue. If not, please re-open the ticket with a description why this doesn't work for you.

Have a nice day!

**Comment by devsnd** [comment]
This is now released in version 1.4.0 and available in pypi.
